modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="understanding-clusterversion-conditiontypes_{context}"]
+= Understanding cluster version condition types
+ 
+The status of the cluster version and its Operators include the condition type, informing you of the health and current state of your cluster version and Operator.
```

Comment:
I don't mind if you stick with "component", but that's what [godocs use][1] to talk about chunks of cluster tooling.  For ClusterVersion, the component is "all the OpenShift Container Platform bits in the cluster", since it's the sitting on the top of that stack.  Underneath there are a number of operands, many of which are operators in their own right, so the `etcd` component, the `kube-apiserver` component, etc. all live within the ClusterVersion component.  Many of those sub-components have ClusterOperators like the `etcd` ClusterOperator to help report the state of their componet up to the cluster-version operator for inclusion in the overall OCP-core component status.  `Upgradeable=False` on the `etcd` ClusterOperator says "the etcd component doesn't think the currrent cluster state is compatible with a 4.(y+1) minor update".  When that gets bubbled up to be `Upgradable=False` on ClusterVersion, it says "the OCP component doesn't think the current state is compatible with a 4.(y+1) minor update".

With that context established, "health and current state of your cluster version and Operator" feels a bit narrow.  We're talking about the ClusterVersion resource, and the cluster-version operator, which your text covers.  But ClusterVersion conditions are also trying to aggregate the state and health of the whole OCP component, which your current text doesn't seem to cover (at least in my reading) as clearly.

[1]: https://github.com/openshift/api/blob/cce310ad2932f6de24491052d506926e484c082c/config/v1/types_cluster_operator.go#L146-L197

Reply:
Sounds good :) updated to: "The Cluster Version Operator (CVO) is responsible for collecting the status conditions of the cluster version and its Operators. This status includes the condition type, informing you of the health and current state of the {product-title} cluster." as the intro